core/scripts/gateway/web_api_trigger/invoke_trigger.go

```diff
@@ -77,30 +76,38 @@ func main() {
 		return
 	}
 
-	payload := `{
-          "trigger_id": "web-api-trigger@1.0.0",
-          "trigger_event_id": "action_1234567890",
-          "timestamp": ` + strconv.Itoa(int(time.Now().Unix())) + `,
-          "topics": ["daily_price_update"],
-					"params": {
-						"bid": "101",
-						"ask": "102"
-					}
-        }
-`
-	payloadJSON := []byte(payload)
+	address := crypto.PubkeyToAddress(key.PublicKey)
+	fmt.Printf("Public Address: %s\n", address.Hex())
+
+	payload := map[string]any{
```

Comment:
does this unmarshal correctly now in the gateway node such that the parameters are persisted?

Reply:
yep params are persisted! was able to test this locally